src/components/Tool/MergeContacts/GetContactDuplicates.graphql

```diff
@@ -1,6 +1,7 @@
 query GetContactDuplicates($accountListId: ID!) {
   # TODO: Eventually needs pagination (Jira issue: MPDX-7642)
-  contactDuplicates(accountListId: $accountListId, first: 50) {
+  contactDuplicates(accountListId: $accountListId, first: 10) {
```

Comment:
Is there a reason we're only fetching the next 10?

Reply:
The angular loads the first 5. I thought 10 was better than 5 or 50 since we are not doing pagination. What do you think?

Reply:
10 is fine